### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by freearhey**
TODO:

- [x] update `tvg-id` and `tvg-name` according to [iptv-org/epg](https://iptv-org.github.io/epg/index.html)
- [x] remove channels from other countries

### CODE REVIEW

**Comment by jkemp814** [comment]
Should I change the rest of those Bumblebee channels to Bumblebee TV? Wasn't sure if you have done it already.

**Comment by jkemp814** [comment]
I went ahead and corrected those, due to my error for not adding 'TV' to the name.

**Comment by freearhey** [comment]
@jkemp814 

> bon appétit
> bspoketv

I have returned the old name.

> Should I change the rest of those Bumblebee channels to Bumblebee TV? Wasn't sure if you have done it already.

Any help would be appreciated.

File: channels/us.m3u [hunk 1]
```diff
@@ -1826,7 +1824,7 @@ https://ign-plex.amagi.tv/hls/amagi_hls_data_ignAAAAAA-ign-plexA/CDN/playlist.m3
 https://ign-plex.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="IGN.us" tvg-name="IGN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ff62vtM.jpg" group-title="News",IGN (1080p)
 https://ign-samsungus.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="WEIU.us" tvg-name="WEIU-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/weiu-color-cobranded-logo-JkTzcMN.png" group-title="General",IL (WEIU) (1080p) [Geo-blocked]
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/us.m3u]: Let's call it `PBS 30 Charleston IL (WEIU-TV)` by now.

[the remark this answers is not in the document]
Comment (jkemp814) [reply channels/us.m3u]: I agree. We know what all these links are, but end users may not have known what WEIU-TV meant, lol.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**